codegen/builtin_fs.go

```diff
@@ -338,6 +333,7 @@ func (f Frontend) Call(ctx context.Context, cln *client.Client, val Value, opts
 				if err != nil {
 					return
 				}
+				fs.SessionOpts = sessionOpts
```

Comment:
This is needed because the frontend solve creates LLB that may need access to a file source specified as a frontend option. For example, a dockerfile context needs the relevant sync provider. Before, this was only working by freak chance because combination of the 5 second delay and bogus session ID caused a session with this `FSSyncProvider` to be available at the same time and buildkit to find it.